go/style.md

```diff
@@ -30,6 +30,6 @@ Performant Go
 =============
 
 1.  When deciding whether to use an interface or a struct: prefer a struct if there will be more than 100 instances. Justification: Interfaces are significantly more expensive in both CPU and memory. Exception: Heterogeneous but identically-used objects (such as AST Nodes) may be best represented with an interface. Single-implementation interfaces are bad for small and many data structures.
-2.  Prefer fewer channels. Justification: Channels are slow and it’s nearly impossible to guarantee you won’t leave zombie goroutines lying around. Channels are threadsafe queues; use them where you might use a queue. Don’t shoehorn them in just because they exist. If you need a high-performance queue, there are alternatives.
+2.  Prefer fewer channels. Justification: Channels [are slow](http://w-dev-blog.appspot.com/posts/2015/03/05/performant-concurrent-queue/index.html) and it’s nearly impossible to guarantee you won’t leave [zombie goroutines](https://groups.google.com/forum/#!topic/golang-nuts/uiySuH8_3Y4) lying around. Channels are threadsafe queues; use them where you might use a queue. Don’t shoehorn them in just because they exist. If you need a high-performance queue, there are [alternatives](https://github.com/Workiva/go-datastructures/blob/master/queue/queue.go).
```

Comment:
I think the dev blog link is only visible to Workiva devs. Is that ok even though this is intended to be a public repo?